cubids/cubids.py

```diff
@@ -410,7 +410,9 @@ def apply_tsv_changes(self, summary_tsv, files_tsv, new_prefix, raise_on_error=T
                 if Path(from_file).exists():
                     # if using datalad, we want to git mv instead of mv
                     if self.use_datalad:
-                        move_ops.append(f"git mv {from_file} {to_file}")
+                        move_ops.append(
+                            f"git mv {os.path.relpath(from_file, str(self.path))} {to_file}"
```

Comment:
relpath fails if from_file isn't in self.path, right? Is that worth checking explicitly beforehand?

Reply:
I realized in retrospect that my git version was kind of old when I first tested, I might revisit this PR again after the hackathon to see if it's necessary.

Reply:
And yes that should fail, but I'm not sure if that fail mode would actually ever occur.